### ISSUE TITLE
Discarding tab with a media player prevents screen/power saver uninhibition

### ISSUE THREAD

**Issue by eugenesan** [rNeomy/auto-tab-discard#401]
TLDR;
If there is at least one discarded tab with media playback, screen/power saver no longer work.

Discarding tab with a media player (before playing, playing, stopped or paused) causes screen/power saver inhibition status to become permanent until the tab or the browser are closed.

Test case:
1. Open a tab with a media player
2. Start a video
3. Observe inhibition status change to "Inhibited" by the browser
4. Stop the video or wait until it finishes
5. Observe browser integration extension detects paused video (unrelated to the system inhibition status)
6. Discard the tab
7. Observe no change in system inhibition and browser integration status
8. Close the tab
9. Observe system inhibition change to Uninhibited and browser integration status change to "No Playback"

The issue exist in Firefox 131 and at least a couple versions back.
Affected media sites are not in exceptions list.
Here are the preferences in case one of the them causes the issue:
```
{
  "chrome.storage.local": {
    "./plugins/blank/core.js": true,
    "./plugins/dummy/core.js": false,
    "./plugins/focus/core.js": false,
    "./plugins/force/core.js": false,
    "./plugins/new/core.js": false,
    "./plugins/next/core.js": false,
    "./plugins/previous/core.js": false,
    "./plugins/trash/core.js": false,
    "./plugins/unloaded/core.js": true,
    "./plugins/youtube/core.js": true,
    "audio": true,
    "battery": false,
    "check-delay": 30000,
    "click": "click.popup",
    "faqs": false,
    "favicon": true,
    "favicon-delay": 500,
    "force.hostnames": [],
    "form": true,
    "go-hidden": false,
    "idle": false,
    "idle-timeout": 300,
    "last-update": 1574387237350,
    "link.context": true,
    "log": false,
    "max.single.discard": 50,
    "memory-enabled": false,
    "memory-value": 60,
    "mode": "time-based",
    "notification.permission": false,
    "number": 1,
    "online": true,
    "page.context": true,
    "paused": false,
    "period": 600,
    "pinned": false,
    "prepends": "💤",
    "release-next-tab": false,
    "simultaneous-jobs": 10,
    "startup-pinned": true,
    "startup-release-pinned": false,
    "startup-unpinned": true,
    "tab.context": true,
    "trash.period": 24,
    "trash.unloaded": false,
    "trash.whitelist-url": [],
    "use-cache": false,
    "whitelist": [],
    "whitelist-url": [],
    "whitelist.session": []
  },
  "localStorage": {
    "click": "popup"
  }
}
```
